ui/Modal/Search.svelte

```diff
@@ -17,7 +17,7 @@
   import { ValidationStatus } from "../src/validation";
 
   import { Icon, Input } from "../DesignSystem/Primitive";
-  import { Remote, TrackToggle } from "../DesignSystem/Component";
+  import { FollowToggle, Remote } from "../DesignSystem/Component";
```

Comment:
Should the component be called `Peer`?

Reply:
This `Remote` component is actually not related to our domain. It already existed in our system and is for remote data fetching. 🤷